config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,13 @@
+Sidekiq.configure_server do |config|
+  config.redis = {
+    host: ENV['REDIS_HOST'],
+    port: ENV['REDIS_PORT'] || '6379'
+  }
+end
+
+Sidekiq.configure_client do |config|
+  config.redis = {
+    host: ENV['REDIS_HOST'],
```

Comment:
also you can use url see docs
https://github.com/mperham/sidekiq/wiki/Using-Redis#using-an-initializer